### PR TITLE
Code Migration

### PR DESCRIPTION
**Pull request by lburgos**

### CODE REVIEW

File: app/uk/gov/hmrc/controllers/HelloWorld.scala [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    _.map(hello => Ok(Json.toJson(hello))).recover(recovery)
+  }
+
+  private def recovery: PartialFunction[Throwable, Result] = {
+    case _ => Status(ErrorInternalServerError.httpStatusCode)(Json.toJson(ErrorInternalServerError))
+  }
+
+}
+
+object SandboxController extends HelloWorld {
```
Comment (AndySpaven) [inline app/uk/gov/hmrc/controllers/HelloWorld.scala]: I think Sandbox is no longer relevant after the Mergeroo

File: app/uk/gov/hmrc/services/HelloWorldService.scala [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    Future.successful(Hello("Hello World"))
+
+  override def fetchApplication(implicit hc: HeaderCarrier): Future[Hello] =
+    Future.successful(Hello("Hello Application"))
+
+  override def fetchUser(implicit hc: HeaderCarrier): Future[Hello] =
+    Future.successful(Hello("Hello User"))
+}
+
+object SandboxService extends HelloWorldService {
```
Comment (AndySpaven) [inline app/uk/gov/hmrc/services/HelloWorldService.scala]: Sandbox - see above

File: resources/public/api/conf/1.0/docs/overview.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+This API allows
```
Comment (AndySpaven) [inline resources/public/api/conf/1.0/docs/overview.md]: This reads oddly

**Comment by lburgos** [comment]
I've addressed the issues identified by Andy

**Pull request merged**